ming_sim/db.py

```
@@ -14158,6 +14159,36 @@ def withdraw_pending_action(self, action_id: int, turn: int) -> bool:
             self.conn.commit()
         return cur.rowcount > 0
 
+    def hold_over_pending_actions(
+        self, turn: int, minister_name: str,
+        action_ids: Optional[Iterable[int]] = None,
+    ) -> int:
+        """#525 留中：点名候选移出 status=pending 活跃集，同表 durable 留中档 (held_over)。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)